client/src/components/Card/Card.js

```diff
@@ -2,10 +2,10 @@ import React from "react";
 import "./Card.css";
 
 const Card = props => {
-  let { size = "large", color, picture, alt } = props;
+  let { size = "large", color, picture, discover } = props;
   return (
     <div
-      className={`Card Card--${color} Card--${size}`}
+      className={`Card Card--${color} Card--${size} Card__discover-${discover}`}
```

Comment:
Note that without a default value, this could break the CSS classname (we could end up with a class name of `Card__discover-` (same actually goes for the `size` value!